static/js/util.js

```diff
@@ -493,6 +494,37 @@ function randDomId() {
     .substr(2, 10);
 }
 
+/**
+ * add or delete statvars from url
+ *
+ * @param {string} dcid of statvar
+ * @param {boolean} add = True, delete = False
+ * @return {string} The line chart URL.
+ */
+
+function updateUrlStatsVar(statvar, add) {
```

Comment:
Minor, but if you rename add to should_add or should_be_added or is_statvar_added, then it's very clear it's a boolean, and line 511 can be very clearly read as: if (is_statvar_added) { ... }

Reply:
done.